### ISSUE TITLE
vagrant-vsphere install fails on vagrant 2.2.1 

### ISSUE THREAD

**Issue by oneasteriskone** [nsidc/vagrant-vsphere#271]
After upgrading to vagrant 2.2.1, I am unable to install the vagrant-vsphere plugin. After downgrading to 2.1.5 it seems to be working again.

Error: 
```
Bundler, the underlying system Vagrant uses to install plugins,
reported an error. The error is shown below. These errors are usually
caused by misconfigured plugin installations or transient network
issues. The error from Bundler is:

conflicting dependencies i18n (~> 0.6.4) and i18n (= 1.1.1)
  Activated i18n-1.1.1
  which does not match conflicting dependency (~> 0.6.4)

  Conflicting dependency chains:
    i18n (= 1.1.1), 1.1.1 activated

  versus:
    vagrant-vsphere (> 0), 0.5.0 activated, depends on
    i18n (~> 0.6.4)
```

**Comment by novacasa** [comment]
got exactly the same - nice to know I am not alone ;)

**Comment by mattiasa** [comment]
I got the same. Downgrade of Vagrant to 2.2.0 solved the issue for me as well.

**Comment by thegreenbear** [comment]
Same here.

**Comment by michael-brandt-cu** [comment]
Fixed with release v1.13.3
